source/includes/steps-3.6-upgrade-sharded-cluster.yaml

```diff
@@ -286,9 +289,12 @@ action:
       On a :program:`mongos` instance, run the
       :dbcommand:`setFeatureCompatibilityVersion` command in the ``admin``
       database:
-   language: javascript
-   code: |
-      db.adminCommand( { setFeatureCompatibilityVersion: "3.6" } )
+
```

Comment:
just an fyi --
to make the code block copyable inside the yaml "action" block, you can just add a copyable: true 


action:
       copyable: true
       language: blah
       code: